pkg/verification/runme.sh

```diff
@@ -143,13 +143,13 @@ do_verification_live() {
      IMAGE_UUID=$(uuidgen | tee /tmp/soft_serial)
      mcopy -o -i /bits/config.img /tmp/soft_serial ::/soft_serial
   fi
-  create_efi_raw "${1:-350}" "$PART_SPEC"
```

Comment:
I think recent PRs have introduced defined names for the sizes; I don't know if that was also the case for this default size in eve/runme.sh.  Please check and update if needed.

Reply:
The equivalent function in pkg/eve/runme.sh line 134: is `do_live()` and the corresponding size in line 146 did not change:
`create_efi_raw "${1:-350}" "$PART_SPEC"`